stdlib/mexpr/pprint.mc

```diff
@@ -313,8 +313,9 @@ lang IntPatPrettyPrint = IntPat
   | PInt t -> int2string t.val
 end
 
-lang CharPatPrettyPring = CharPat
-  -- TODO
+lang CharPatPrettyPrint = CharPat
+  sem getPatStringCode (indent : Int) =
+  | PChar t -> [head "'", t.val, head "'"]
```

Comment:
You can use `'\''` instead of `head "'"`.

Reply:
Thanks, fixed (also for `CharPrettyPrint`).